web/src/app/settings/components/Settings/expected-members.ts

```diff
@@ -0,0 +1,11 @@
+export enum SettingMembers {
+  nameHighPriority = 'nameHighPriority',
+  nameMediumPriority = 'nameMediumPriority',
+  nameLowPriority = 'nameLowPriority',
+  colorHighPriority = 'colorHighPriority',
+  colorMediumPriority = 'colorMediumPriority',
+  colorLowPriority = 'colorLowPriority',
+  intervalHighPriority = 'intervalHighPriority',
+  intervalMediumPriority = 'intervalMediumPriority',
+  intervalLowPriority = 'intervalLowPriority'
+}
```

Comment:
https://www.typescriptlang.org/docs/handbook/enums.html enum name should have capitalized names but this is not a big issue